custom_functions/mark_evidence.py

```diff
@@ -1,10 +1,10 @@
-def mark_evidence(container=None, object_id=None, content_type=None, **kwargs):
+def mark_evidence(container=None, input_object=None, content_type=None, **kwargs):
     """
     Mark an object as Evidence in a container
     
     Args:
         container (CEF type: phantom container id): Container ID or Container Object
-        object_id (CEF type: *): Id of object to be added - artifact id, note id, etc.
+        input_object (CEF type: *): Id or object to be added - artifact id, note id, etc.
```

Comment:
I think users will need a bit more here to make it clear how to use the `results` object. It is not clear that the `actionrun` type maps to keyword_arguments:results. Also, the results object will fail if there is another block between the action run and the custom function, including if there is a join block. Maybe something like this:

input_object (CEF type: *): The object to mark as evidence. This could be a vault_id, artifact_id, note_id, or if the previous playbook block is an action then "keyword_argument:results" can be used with the content_type "actionrun"

Reply:
Also, I think container needs to be removed from the content_type options (I left it out above) because there seems to be no way to pass the literal string "container" into the custom function. It just ends up as a blank string.